dashboard/app/jobs.go

```diff
@@ -195,11 +195,12 @@ func addTestJob(c context.Context, args *testJobArgs, now time.Time) (string, er
 
 func checkTestJob(c context.Context, bug *Bug, bugReporting *BugReporting, crash *Crash,
 	repo, branch string) string {
-	isBootError := func(crash *Crash) bool {
-		return strings.Contains(crash.Title, "boot error:")
+	testWithoutRepro := func(crash *Crash) bool {
```

Comment:
Not sure why it was a function in the first place, but can't we just use a bool var here?
```
needRepro := !strings.Contains(crash.Title, "boot error:") &&
			!strings.Contains(crash.Title, "test error:")
```